polkadot/node/core/pvf/src/execute/queue.rs

```diff
@@ -811,6 +835,7 @@ mod tests {
 		ExecuteJob {
 			artifact: ArtifactPathId { id: artifact_id(0), path: PathBuf::new() },
 			exec_timeout: Duration::from_secs(10),
+			exec_deadline: None,
```

Comment:
I think we should have a test with deadline exceeded as well.

Reply:
True